### ISSUE TITLE
Yared, are you the only one who contributed here?

### ISSUE THREAD

**Issue by natnaelargaw** [Yared-betsega/monte-carlo-cfr-poker#2]

**Comment by Yared-betsega** [comment]
No, we both investigated Poker together. But we were working on my computer in one setting. We focused more on understanding the self-play, the Nash equilibrium, and solving the game. So we wrote the code on my PC.
